app/controllers/ticket_purchases_controller.rb

```diff
@@ -30,6 +31,18 @@ def index
 
   private
 
+  def count_purchased_registration_tickets
+    purchases = params[:tickets][0]
+    purchased_registration_tickets = 0
+    @conference.tickets.for_registration.each do |registration_ticket|
+      purchased_registration_tickets += purchases[registration_ticket.id.to_s].to_i
+    end
```

Comment:
no, I was trying to look for the tickets using the ids from `purchased_tickets`, so `purchased_tickets.keys`, sorry:

``` Ruby
purchased_tickets = params[:tickets][0]
tickets = Ticket.find(purchased_tickets.keys).where(registration_ticket: true)
if tickets.count > 1 || purchases[tickets.first.id] != '1'
  redirect_to conference_tickets_path(@conference.short_title),
                  error: 'You cannot buy more than one registration tickets.'
end
```